### ISSUE TITLE
--force flag language is ambiguous 

### ISSUE THREAD

**Issue by zackproser** [gruntwork-io/cloud-nuke#346]
```
 --force                        Skip nuke confirmation prompt. WARNING: this will automatically delete all resources without any confirmation
```

This language, specifically `automatically delete all resources` - is unclear - what happens if you run cloud-nuke with a config file, specifying that only specifically named Lambda functions, for example, should be deleted?

It's not clear if the selections made in the config file will be honored, or not, if you pass the `--force` flag in this scenario. 

To be clear, the actual behavior is that cloud-nuke _will_ honor the selections made in the config file, but this could be made more clear in the help text / description for the `--force` flag itself.

TLDR: we need to clarify that passing the `--force` flag will only skip the nuke confirmation step - and replace it with a 10 second countdown. It will not modify the selections made by other flags or a config file.
